### PR TITLE
Use latest eve

### PR DESCRIPTION
**Pull request by amagdas**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:  Any plans to upgrade other frameworks? Status shows they are outdated.

**Comment by amagdas** [comment]
Yup, I'll look into it. Some of them can't be upgrade yet because of dependencies.

File: server/requirements.txt [hunk 1]
```diff
@@ -3,7 +3,5 @@ honcho==0.6.6
 python3-ldap==0.9.8.4
 behave==1.2.5
 wooper==0.4.2
-boto3==1.0.0
 
-git+git://github.com/superdesk/eve@sync-develop-24-june#egg=Eve==0.6.0-dev
```
Comment (petrjasek) [inline server/requirements.txt]: I think we still need this one, you can't set via setup in `superdesk-core` a git version of `superdesk/eve` can you?

Comment (amagdas) [inline server/requirements.txt]: Hmm, works fine locally...

**Pull request closed without merging**